atlasaction/action.go

```diff
@@ -40,3 +54,63 @@ func MigrateApply(ctx context.Context, client *atlasexec.Client, act *githubacti
 	act.Infof("Run complete: +%v", run)
 	return nil
 }
+
+// MigratePush runs the Github Action for "ariga/atlas-action/migrate/push"
+func MigratePush(ctx context.Context, client *atlasexec.Client, act *githubactions.Action) error {
+	ghContext, err := createContext(act)
+	if err != nil {
+		return fmt.Errorf("failed to read github metadata: %w", err)
+	}
+	buf, err := json.Marshal(ghContext)
+	if err != nil {
+		return fmt.Errorf("failed to create MigratePushParams: %w", err)
+	}
+	params := &atlasexec.MigratePushParams{
+		Name:      act.GetInput("dir-name"),
+		DirURL:    act.GetInput("dir"),
+		DevURL:    act.GetInput("dev-url"),
+		Context:   string(buf),
+		ConfigURL: act.GetInput("config"),
+		Env:       act.GetInput("env"),
+	}
+	_, err = client.MigratePush(context.Background(), params)
```

Comment:
```suggestion
	msg, err := client.MigratePush(context.Background(), params)
```
and use it as an output. This makes it easier to consume the result for users.